public/404.html

```diff
@@ -1,67 +0,0 @@
-<!DOCTYPE html>
```

Comment:
Keep these files just as a backup that the app does not even start. We need these static ones too.